cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java

```diff
@@ -90,20 +92,30 @@ public boolean isSimpleType() {
    * Logical type TIMESTAMP_MICROS relies on LONG as underlying primitive type
    * Logical type TIME_MILLIS relies on INT as underlying primitive type
    * Logical type TIME_MICROS relies on LONG as underlying primitive type
+   * Logical type DECIMAL relies on BYTES as underlying primitive type
    *
    * For example, the json schema for logicaltype date will be as below:
    * {
    *   "type" : "int",
    *   "logicalType" : "date"
    * }
    *
+   * Example schema for logicaltype decimal will be as below:
+   * {
+   *   "type" : "bytes",
+   *   "logicalType" : "decimal",
+   *   "precision" : 4,
+   *   "scale" : 2
+   * }
+   * where precision is a required field and scale is an optional field.
```

Comment:
Can you add some information about what precision and scale are with an example?